recipe/meta.yaml

```diff
@@ -22,35 +22,36 @@ requirements:
   run:
     - python >=3.8,<3.11
     - pip
-    - coiled
-    - numpy ==1.21.6
-    - pandas ==1.4.2
+    - coiled >=0.2.41
+    - numpy ==1.23.3
+    - pandas ==1.5.1
     - dask =={{ dask_version }}
     - distributed =={{ distributed_version }}
-    - fsspec ==2022.5.0
-    - s3fs ==2022.5.0
-    - gcsfs ==2022.5.0
-    - pyarrow ==8.0.0
-    - jupyterlab ==3.4.2
-    - dask-labextension ==5.2.0
-    - lz4 ==4.0.0
-    - ipywidgets ==7.7.0
+    - fsspec ==2022.10.0
+    - s3fs ==2022.10.0
+    - gcsfs ==2022.10.0
+    - pyarrow ==9.0.0
+    - jupyterlab ==3.4.8
+    - dask-labextension ==5.3.0
+    - lz4 ==4.0.2
+    - ipywidgets ==7.7.2
     - numba ==0.56.3
-    - scikit-learn ==1.1.1
+    - scikit-learn ==1.1.3
     - ipycytoscape ==1.3.3
     - click ==8.1.3
-    - xarray ==2022.3.0
-    - zarr ==2.11.3
-    - msgpack-python ==1.0.3
-    - cloudpickle ==2.1.0
+    - xarray ==2022.10.0
+    - zarr ==2.13.3
+    - msgpack-python ==1.0.4
+    - cloudpickle ==2.2.0
     - tornado ==6.1
-    - toolz ==0.11.2
-    - python-blosc ==1.10.2
+    - toolz ==0.12.0
+    - python-blosc ==1.10.6
     - zict ==2.2.0
-    - xgboost ==1.6.1
+    - xgboost ==1.6.2
     - dask-ml ==2022.5.27
     - openssl >1.1.0g
     - pynvml ==11.4.1
+    - optuna ==3.0.3
```

Comment:
```suggestion
    - bokeh ==2.4.3
```

Reply:
Thanks, @crusaderky. I will commit it when this first CI run finishes.